packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -58,50 +59,54 @@ const cookieStorage = onClient ? new CookieStorage() : new Map()
 const DATA_MAP: AuthDataMap = {
     access_token: {
         storage: localStorage,
-        key: 'access_token'
+        key: 'access_token',
     },
     customer_id: {
         storage: localStorage,
-        key: 'customer_id'
+        key: 'customer_id',
     },
     usid: {
         storage: localStorage,
-        key: 'usid'
+        key: 'usid',
     },
     enc_user_id: {
         storage: localStorage,
-        key: 'enc_user_id'
+        key: 'enc_user_id',
     },
     expires_in: {
         storage: localStorage,
-        key: 'expires_in'
+        key: 'expires_in',
     },
     id_token: {
         storage: localStorage,
-        key: 'id_token'
+        key: 'id_token',
     },
     idp_access_token: {
         storage: localStorage,
-        key: 'idp_access_token'
+        key: 'idp_access_token',
     },
     token_type: {
         storage: localStorage,
-        key: 'token_type'
+        key: 'token_type',
     },
     refresh_token_guest: {
         storage: cookieStorage,
         key: 'cc-nx-g',
         callback: () => {
             cookieStorage.delete('cc-nx')
-        }
+        },
     },
     refresh_token_registered: {
         storage: cookieStorage,
         key: 'cc-nx',
         callback: () => {
             cookieStorage.delete('cc-nx-g')
-        }
-    }
+        },
+    },
+    site_id: {
+        storage: cookieStorage,
+        key: 'cc-site-id',
```

Comment:
The ticket mentions about namespacing the token. Do we still want to do that?

Without namespacing it, I guess the consequence would be that when you switch back to a previously-visited site, you would lose your basket progress?

Reply:
this cookie is used as the "namespacer". It's not realistic to store tokens for every single site as we are bound by the size limitation of Cookie/Localstorage. A customer could have 100+ sites and we can't keep them all.

You are right that customer will lose the basket progress unfortunately.